packages/material-ui-lab/src/ToggleButton/ToggleButton.test.js

```diff
@@ -22,6 +22,10 @@ describe('<ToggleButton />', () => {
     classes = getClasses(<ToggleButton value="classes">Hello World</ToggleButton>);
   });
 
+  after(() => {
+    mount.cleanUp();
```

Comment:
Spotted some missing cleanups. Or rather the linter. Will introduce the rule in a separate PR. It's probably overkill since those things should be spotted in review. Lint rule might be to brittle.

Reply:
It might be overkill.

Reply:
Nope it was causing test failures because DOM nodes were retained in subsequent tests.

Reply:
I would go for adding cleanUp to everything

Reply:
That wouldn't work in its current form though as far as I know. `cleanUp` removes the DOM node were the mounted components are attached to. This would mean we would have to `createMount` after every `cleanUp`.

It would work with https://github.com/mui-org/material-ui/pull/14987#discussion_r269201103

Reply:
That change looks good to me 👍